src/analysis/normed_space/inner_product.lean

```diff
@@ -1374,6 +1382,56 @@ linear_map.mk_continuous
 
 end norm
 
+section bessels_inequality
+
+variables {ι: Type*} (x : E) {v : ι → E}
+
+lemma bessel_finite {s : finset ι} (hv : orthonormal 𝕜 v) : ∑ i in s, ∥ ⟪v i, x⟫ ∥^2 ≤ ∥x∥ ^2 :=
```

Comment:
The convention is to make the name descriptive of the contents, and then put the "human" name in a docstring above.  Here, perhaps `orthonormal.sum_inner_products_le` would work (and `orthonormal.tsum_inner_products_le`, `orthonormal.inner_products_summable` for the others) -- putting them in the namespace `orthonormal` allows for the use of projection notation with the variable `hv`, as we were discussing yesterday.